modules/gdscript/gdscript_function.h

```diff
@@ -107,7 +107,7 @@ class GDScriptDataType {
 
 				Object *obj = p_variant.get_validated_object();
 				if (!obj) {
-					return false;
+					return true;
```

Comment:
Might be worth adding a comment to clarify why this is considered true.

Reply:
I added a comment there to explain why.